### PR TITLE
LIP-25: Replace 'abstract' section with non-technical LIP introduction

### PR DESCRIPTION
**Pull request by kyriediculous**

### CODE REVIEW

File: LIPs/LIP-25.md [hunk 1]
```diff
@@ -8,9 +8,14 @@ created: 2020-06-09
 discussions-to: https://github.com/livepeer/LIPs/issues/25
 ---
 
-## Abstract
+## Introduction
```
Comment (yondonfu) [inline LIPs/LIP-25.md:11]: Let's keep the "Abstract" title in accordance with the [LIP-X](https://github.com/livepeer/LIPs/blob/master/LIP-X.md) template.

File: LIPs/LIP-25.md [hunk 2]
```diff
@@ -8,9 +8,14 @@ created: 2020-06-09
...
+
+This proposal is the next step towards **decentralising ownership over the Livepeer protocol** and fulfills the second milestone of the [Livepeer Governance Roadmap](https://medium.com/livepeer-blog/livepeer-governance-roadmap-proposal-69a6e9e33f80). It transfers ownership of the Livepeer protocol to a smart contract which in a later milestone can be owned by a binding voting system. It is a clear step forward in transferring control over the Livepeer protocol from the core team to all stakeholders (tokenholders, orchestrators, broadcasters, ... ).
+
+This LIP offers **complete transparency** changes being madeto the Livepeer protocol and the ability for the community or individual stakeholders to object to changes or exit the protocol if they don't agree with the changes being made or if changes are executed in bad faith. This new transparancy properties further **strengthens the social contract** that is implicitly established between the stakeholders and the core team executing the changes.
```
Comment (yondonfu) [inline LIPs/LIP-25.md:15]: "complete transparency changes" -> "complete transparency around changes"

Comment (yondonfu) [inline LIPs/LIP-25.md:15]: "madeto" -> "made to"

Comment (yondonfu) [inline LIPs/LIP-25.md:15]: "This new transparency properties" -> "These new transparency properties"

**Review by yondonfu — CHANGES_REQUESTED** [review]

File: LIPs/LIP-25.md (hunk 2 above)
Comment (kyriediculous) [reply LIPs/LIP-25.md]: fixed in 7cb8845

Comment (kyriediculous) [reply LIPs/LIP-25.md]: fixed in 7cb8845

Comment (kyriediculous) [reply LIPs/LIP-25.md]: fixed in 7cb8845

File: LIPs/LIP-25.md (hunk 1 above)
Comment (kyriediculous) [reply LIPs/LIP-25.md]: fixed in 7cb8845

**Pull request merged**